src/main/java/com/wolApi/wolApi/AppSettings.java

```diff
@@ -0,0 +1,10 @@
+package com.wolApi.wolApi;
+
+public class AppSettings {
```

Comment:
This is good, but will likely have to be changed in the future if we are to add support for different languages/bible translations

Reply:
Yeah this was just temporary refactoring